app/controllers/bank_accounts_controller.rb

```diff
@@ -0,0 +1,53 @@
+class BankAccountsController < ApplicationController
+  def index
+    if current_user.banker ==true || current_user.staff == true
```

Comment:
For boolean columns, you can use methods suffixed with question marks, e.g. `current_user.banker? || current_user.staff?`, which I think looks cleaner, and more inline with the Ruby conventions.

Reply:
Changed to match your suggestion